index.html

```diff
@@ -41,6 +40,151 @@ <h1>I'm Tash <br> Glad to see you!</h1>
+			<div class="langs2">
+				<button class="btn12" type="submit">html</button>
+				<button class="btn22" type="submit">css</button>
+				<button class="btn32" type="submit">javaScript</button>
+			</div>
+			<button class="btn42" type="submit">See Project</button>
+			</div>
+
+			<div class="works3">
+			<img class="snap3" src="images/Snapshoot3.png" alt="">
+			<p class="tonic3">Tonic</p>
+
+			<div class="grid-container3">
+				  <div class="grid-item13"><p>CANOPY</p></div>
+				  <div class="grid-item23"><p>.</p></div>
+				  <div class="grid-item33"><p>Back End Dev</p></div>
+				  <div class="grid-item43"><p>.</p></div>
+				  <div class="grid-item53"><p>2015</p></div>
+			</div>
+			<p class="day3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs3">
+				<button class="btn13" type="submit">html</button>
+				<button class="btn23" type="submit">css</button>
+				<button class="btn33" type="submit">javaScript</button>
+			</div>
+			<button class="btn43" type="submit">See Project</button>
+			</div>
+
+			<div class="works4">
+			<img class="snap4" src="images/Snapshoot4.png" alt="">
+			<p class="tonic4">Multi-Post Stories</p>
+
+			<div class="grid-container4">
+				  <div class="grid-item14"><p>CANOPY</p></div>
+				  <div class="grid-item24"><p>.</p></div>
+				  <div class="grid-item34"><p>Back End Dev</p></div>
+				  <div class="grid-item44"><p>.</p></div>
+				  <div class="grid-item54"><p>2015</p></div>
+			</div>
+			<p class="day4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+			<div class="langs3">
+				<button class="btn14" type="submit">html</button>
+				<button class="btn24" type="submit">css</button>
+				<button class="btn34" type="submit">javaScript</button>
+			</div>
+			<button class="btn44" type="submit">See Project</button>
+			</div>
+
+		</section>
+
+		<div class="div-class">
+
+		<section class="about-myself">
+
+      <div class="desk-about">
+        <h1 class="aboutmy">About <br> Myself</h1>
+        <p class="am">I’m a software developer! I can help you build a product , feature or website Look through some of my
+          work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+          contact me.</p>
+        <P class="connect">LET'S CONNECT</P>
+        <ul class="contact-icon">
+          <li><a href="#" class="g"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+          <li><a href="#" class="l"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+          <li><a href="#" class="a"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+          <li><a href="#" class="t"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#" class="m"><i class="fa-solid fa-m" aria-hidden="true"></i></a></li>
+        </ul>
+        <button class="resume">Get My Resume</button>
```

Comment:
- [ ] Kindly add the button interaction styling like the hover and press for all of your buttons.